elastic/kibana
# [Index Management] Accept array of strings or string in copy_to field

Fixes: https://github.com/elastic/kibana/issues/146360

## Summary

Updates the schema type of `copy_to` to accept  both a string or an array of strings.

**How to test**
1. Start elasticsearch with `yarn es snapshot --license=trial` and kibana with `yarn start`
2. Navigate to `Stack Management` -> `Index Management` -> `Component Templates`, and create a new template with a any name and goto mappings (step 3)
3. Click load json and add the following json mapping and click load
```
{
  "properties": {
    "text1": {
      "copy_to": ["field1", "field2"],
      "type": "text"
    },
    "field1": { "type": "text" },
    "field2": { "type": "text" }
  }
}
```
5. Verify that:
    * It adds the correct mappings to the UI
    * Editing `text1` mapping and clicking show advanced settings, the `copy to group field` should show `field1,field2`
    * The template saves correctly
